src/jira/client/jira-client-issue-key-helper.test.ts

```diff
@@ -0,0 +1,667 @@
+import {
```

Comment:
These are all new :sigh: but ive isolated each function into its own describe block so its pretty easy to follow IMO but will be having a re-review over this too